config/label_templates.yml

```diff
@@ -17,6 +17,12 @@ default_printer_type_names:
   :tube_rack: 96 Well Plate
   :tube: 1D Tube
 
+plate_a:
+  :label_class: Labels::PlateLabel
+  :printer_type: 96 Well Plate
+  :pmb_template: sqsc_96plate_label_template_code39
+  :sprint_template: plate_96.yml.erb
+
```

Comment:
plate_a is also used as keys in the default sections of the label_templates configuration, which are picked up by label classes in their default methods. However, this is actually a good idea, grouping them together under a label_template and use in purpose config.